dittopresenceviewer/src/main/java/ditto/live/dittopresenceviewer/PresenceViewerScreen.kt

```diff
@@ -68,15 +51,103 @@ class PresenceViewModel : ViewModel() {
             peersObserver?.close()
             value?.let { ditto ->
                 peersObserver = ditto.presence.observe { graph ->
-                    presenceJSON.value = graph.json()
+                    Log.d(TAG,"network " + graph.json())
+                    updateNetwork(graph.json()) {}
```

Comment:
Update network directly, no need to use a mutable state here since we aren't rendering the presenceJSON directly in the UI, we are sending it to javascript.